libs/platform-info/src/main/kotlin/net/corda/libs/platform/PlatformVersion.kt

```diff
@@ -4,4 +4,5 @@ enum class PlatformVersion(val value: Int) {
     CORDA_5_0(50000),
     CORDA_5_1(50100),
     CORDA_5_2(50200),
+    CORDA_JSON_BLOB_HEADER(50201)
```

Comment:
I wonder if we should additionally have a `CORDA_5_2_1(50201)` as well to make it obvious what was the "first" release with it?  Or does there have to be a 1-1 mapping?

This would get complicated if we released 5.3 first!

Reply:
Our platform version gets complicated if we release 5.3 without a feature that is in 5.2.1 - the simple >= check then wouldn't work.

Reply:
I don't think this has to be a 1:1 mapping, they are only used explicitely, so we could do something like
```suggestion
    CORDA_5_2_1(50201)
    CORDA_JSON_BLOB_HEADER(CORDA_5_2_1.value)
```